packages/experimental/TabList/src/Tab/useTab.ts

```diff
@@ -49,7 +63,12 @@ export const useTab = (props: TabProps): TabInfo => {
     [changeSelection],
   );
 
-  /* We use the componentRef of the currently selected tabsItem to maintain the default tabbable
+  const accessibilityActionsProp = React.useMemo(
```

Comment:
Since we're allowing the user to pass their own value for `accessibilityActions`, we should also call their `onAccessibilityAction` prop. Similar to: https://github.com/microsoft/fluentui-react-native/blob/edebdac97d38c85002ccee7f074f288dfd629c0c/packages/components/Menu/src/MenuItemCheckbox/useMenuItemCheckbox.ts#L103C9-L103C35

I'd also check if we're disabled before we call 'Select' in that function.